src/org/opendatakit/briefcase/util/ExportToCsv.java

```diff
@@ -134,6 +136,46 @@ public boolean doAction() {
 
     File[] instances = instancesDir.listFiles();
 
+		// Sorts the instances by the submission date. If no submission date, we
+		// assume it to be latest.
+		Arrays.sort(instances, new Comparator<File>() {
+			HashMap<File, String> submissionDateToFileMap = new HashMap<File, String>();
+			File submission1, submission2;

```

Comment:
It's good practice to declare variables in the tightest scope possible. In this case, `submission1` and `submission2` should just be declared when they are first initialized (`File submission1 = new File(f1, "submission.xml");`)

`submissionDate1String` and `submissionDate2String` can be declared above the conditionals that set them. 

`submissionDate1` and `submissionDate2` can be declared where they are initialized.